src/Libraries/Microsoft.AspNetCore.HeaderParsing/README.md

```diff
@@ -1 +1,130 @@
-README
+# Microsoft.AspNetCore.HeaderParsing
+
+This package provides services to for strongly typed header parsing and value caching.
+
+In particular:
+
+- There are many different headers with subtle format differences that are hard to parse correctly in all cases. This package provides predefined parsers for commonly used headers.
+- Your application may parse the same header value multiple times per request. This package automatically caches the parsed values of these headers to make your application faster.
+- This package also provides logging and metering of bad header values.
+
+## Getting started
+
+### Install the package
+
+From the command-line:
+
+```dotnetcli
+dotnet add package Microsoft.AspNetCore.HeaderParsing
+```
+
+Or directly in the C# project file:
+
+```xml
+<ItemGroup>
+  <PackageReference Include="Microsoft.AspNetCore.HeaderParsing" Version="*" />
```

Comment:
Don't encourage bad habits
```suggestion
  <PackageReference Include="Microsoft.AspNetCore.HeaderParsing" Version="[CURRENTVERSION]" />
```

Reply:
I had no better idea :/ I noticed that @joperezr cheated and only mentioned the dotnet-cli way ;) I think we should converge and use the same pattern everywhere. At least this starts the discussion.